public_html/formCreator/templates/fieldSettingsForm.php

```diff
@@ -133,16 +133,24 @@
                     {local var="formsOptions"}
                 </select>
 
-                <label for="fieldSettings_choices_fieldSelect">
-                    Select a Field
-                </label>
-                <select class="input-block-level" id="fieldSettings_choices_fieldSelect" name="fieldSettings_choices_fieldSelect" data-bindModel="choicesField">
-                </select>
+                <div class="fieldsContainer hidden">
+                  <label for="fieldSettings_choices_fieldSelect">
+                      Select a Field
+                  </label>
+
+                  <select class="input-block-level" id="fieldSettings_choices_fieldSelect" name="fieldSettings_choices_fieldSelect" data-bindModel="choicesField">
+                  </select>
+
+                  <button type="button" name="addAdditionalLinkedFields" id="choicesExtraFields_button" class="extraFieldsBtn"> Add Additional Field Options </button>
+
+                  <div class="additionalFieldContainer"></div>
+                </div>
```

Comment:
This block does not look like it pertains to this update (public and oai check boxes). If it doesn't, please revert this change.